jobs/go/sdk/job-service/job-service.go

```diff
@@ -0,0 +1,221 @@
+limitations under the License.
+*/
+
+package main
+
+/*
+dapr run --app-id maintenance-scheduler --app-port 5200 --dapr-http-port 5280 --dapr-grpc-port 5281 --scheduler-host-address=127.0.0.1:50006 -- go run .
+*/
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/dapr/go-sdk/service/common"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	daprc "github.com/dapr/go-sdk/client"
+	daprs "github.com/dapr/go-sdk/service/grpc"
+)
+
+type App struct {
+	daprClient daprc.Client
+}
+
+type DroidJob struct {
+	Name    string `json:"name"`
+	Job     string `json:"job"`
+	DueTime string `json:"dueTime"`
+}
+
+type JobData struct {
+	Droid string `json:"droid"`
+	Task  string `json:"Task"`
+}
+
+var jobNames = []string{"R2-D2", "C-3PO", "BB-8"}
+
+var app App
+
+func main() {
+
+	appPort := os.Getenv("APP_PORT")
+	if appPort == "" {
+		appPort = "6200"
+	}
+
+	//Create new Dapr client
+	daprClient, err := daprc.NewClient()
+	if err != nil {
+		panic(err)
+	}
+	defer daprClient.Close()
+
+	app = App{
+		daprClient: daprClient,
+	}
+
+	// Create a new Dapr service
+	server, err := daprs.NewService(":" + appPort)
+	if err != nil {
+		log.Fatalf("failed to start server: %v", err)
+	}
+
+	// Creates handlers for the service
+	if err := server.AddServiceInvocationHandler("scheduleJob", scheduleJob); err != nil {
+		log.Fatalf("error adding invocation handler: %v", err)
+	}
+
+	if err := server.AddServiceInvocationHandler("getJob", getJob); err != nil {
+		log.Fatalf("error adding invocation handler: %v", err)
+	}
+
+	if err := server.AddServiceInvocationHandler("deleteJob", deleteJob); err != nil {
+		log.Fatalf("error adding invocation handler: %v", err)
+	}
+
+	// Register job event handler for all jobs
+	for _, jobName := range jobNames {
+		if err := server.AddJobEventHandler(jobName, handleJob); err != nil {
+			log.Fatalf("failed to register job event handler: %v", err)
+		}
+		fmt.Println("Registered job handler for: ", jobName)
+	}
+
+	fmt.Println("Starting server on port: " + appPort)
+	if err = server.Start(); err != nil {
+		log.Fatalf("failed to start server: %v", err)
+	}
+}
+
+// Handler that schedules a DroidJob
+func scheduleJob(ctx context.Context, in *common.InvocationEvent) (out *common.Content, err error) {
+
+	if in == nil {
+		err = errors.New("no invocation parameter")
+		return
+	}
+
+	droidJob := DroidJob{}
+	err = json.Unmarshal(in.Data, &droidJob)
+	if err != nil {
+		fmt.Println("failed to unmarshal job: ", err)
+		return nil, err
+	}
+
+	jobData := JobData{
+		Droid: droidJob.Name,
+		Task:  droidJob.Job,
+	}
+
+	content, err := json.Marshal(jobData)
+	if err != nil {
+		fmt.Printf("Error marshalling job content")
+		return nil, err
+	}
+
+	// schedule job
+	job := daprc.Job{
+		Name:    droidJob.Name,
+		DueTime: droidJob.DueTime,
+		Data: &anypb.Any{
+			Value: content,
+		},
+	}
+
+	err = app.daprClient.ScheduleJobAlpha1(context.Background(), &job)
```

Comment:
```suggestion
	err = app.daprClient.ScheduleJobAlpha1(ctx, &job)
```
Avoiding creating an empty context and instead passing the job context